distribution/lib/Standard/Base/0.0.0-dev/src/Error/Time_Error.enso

```diff
@@ -0,0 +1,21 @@
+import project.Data.Text.Text
+
+type Time_Error
+
+    ## UNSTABLE
+
+       An error produced while working with time- and date-related methods.
+
+       Arguments:
+       - error_message: The message for the error.
+    Error error_message
```

Comment:
I don't like that we are inconsistent with doc comments - some are attached to the _type_ while some are attached to a _constructor_.

IIRC the latter is needed to make these docs properly show up in the searcher (unless something has changed), while the former looks much better in text mode.

Regardless of the choice, we should at least be consistent here.

Reply:
Thank you